website/scripts/launchPlatform.php

```diff
@@ -17,14 +17,14 @@
   //Check if remotelaunch parameter is selected. 
   if (!isset($_GET['remotelaunch'])){
     // Switch to main.html
-    //header("Location: ../main.html");
-    exit;
+    header("Location: ../main.html");
```

Comment:
@dan-du-car you have commented this out on your PR https://github.com/usdot-fhwa-stol/carma-web-ui/pull/54/files. Why are you putting this back in again? I thought your index is going to do the job to redirect instead of this.

Reply:
right, this exit is just stop execute the rest of the bash. I comment out the redirect